internal/tlsutil/cfg.go

```diff
@@ -56,3 +62,77 @@ func ClientConfig(opts Options) (cfg *tls.Config, err error) {
 	cfg = &tls.Config{InsecureSkipVerify: opts.InsecureSkipVerify, Certificates: []tls.Certificate{*cert}, RootCAs: pool}
 	return cfg, nil
 }
+
+func ReadCertFromSecDir(host string) (opts Options, err error) {
+	if runtime.GOOS == "windows" || runtime.GOOS == "unix" {
+		fmt.Printf("%v OS not supported for this oci pull. Contact your administrator for more information !!!", runtime.GOOS)
```

Comment:
Why should the user have to contact their administrator for Helm support?

Reply:
Yeah, you are right user don't need to contact their administrator because our user message is sufficient.
I will remove that message.
Thanks.